src/AppInstallerCLICore/Workflows/DownloadFlow.cpp

```diff
@@ -183,7 +183,7 @@ namespace AppInstaller::CLI::Workflow
     void CheckForExistingInstaller(Execution::Context& context)
     {
         const auto& installer = context.Get<Execution::Data::Installer>().value();
-        if (installer.InstallerType == InstallerTypeEnum::MSStore)
+        if (installer.EffectiveInstallerType() == InstallerTypeEnum::MSStore)

```

Comment:
[This is the correct usage here just calling this out if it isn't already done...]
As `MSStore` is not an installer type that actually has an installer file (to us), it should not be a valid nested type.  Is that already handled elsewhere?

Reply:
The only check I had was from manifest validation. I added another check prior to the install flow to verify that if the NestedInstallertype is valid if the BaseInstallertype is an archive. Outputs the appropriate error message if NestedInstallertype is not supported.